requirements.txt

```diff
@@ -1,7 +1,7 @@
 Cython>=0.25.2,<0.26
 Twisted>=16.6.0,<16.7
 Jinja2>=2.8,<2.9
-Pillow>=3.4.2,<3.5
+Pillow>=3.4.2,<=4.0
```

Comment:
should be starting from 4.0, shouldn't it? You are using new method, so should be new version range